src/parser/decompile.sh

```diff
@@ -6,11 +6,22 @@ $DECOMPILER_PATH/Decompiler.exe -i "$DEADLOCK_PATH\game\citadel\pak01_dir.vpk" -
 $DECOMPILER_PATH/Decompiler.exe -i "$DEADLOCK_PATH\game\citadel\pak01_dir.vpk" --output "decompiled-data" --vpk_filepath "scripts/generic_data.vdata_c" -d
 
 mkdir -p "decompiled-data\localizations"
-cp "$DEADLOCK_PATH\game\citadel\resource\localization\citadel_gc\citadel_gc_english.txt" "decompiled-data\localizations\citadel_gc_english.txt"
-python3 parsers/localization.py "decompiled-data\localizations\citadel_gc_english.txt" "decompiled-data\localizations\citadel_gc_english.json"
 
-cp "$DEADLOCK_PATH\game\citadel\resource\localization\citadel_mods\citadel_mods_english.txt" "decompiled-data\localizations\citadel_mods_english.txt"
-python3 parsers/localization.py "decompiled-data\localizations\citadel_mods_english.txt" "decompiled-data\localizations\citadel_mods_english.json"
+# Define an array of folders to parse
+#folders=("citadel_attributes" "citadel_dev" "citadel_gc" "citadel_generated_vo" "citadel_heroes" "citadel_main" "citadel_mods") # All folders
+folders=("citadel_attributes" "citadel_gc" "citadel_heroes" "citadel_main" "citadel_mods") # All folders but voice lines and dev for now
 
-cp "$DEADLOCK_PATH\game\citadel\resource\localization\citadel_heroes\citadel_heroes_english.txt" "decompiled-data\localizations\citadel_heroes_english.txt"
-python3 parsers/localization.py "decompiled-data\localizations\citadel_heroes_english.txt" "decompiled-data\localizations\citadel_heroes_english.json"
+# Loop through each folder in the array
+for folder in "${folders[@]}"; do
+  # Construct the source path using DEADLOCK_PATH and folder name
+  src_path="$DEADLOCK_PATH/game/citadel/resource/localization/$folder"
+  
+  # Construct the destination path by replacing "citadel_" prefix with ""
+  dest_folder_name="${folder#citadel_}"
+  dest_path="decompiled-data/localizations/$dest_folder_name"
+  mkdir -p $dest_path
+
+  # Run the Python script to parse the folder
+  python3 localization_parser.py "$src_path" "$dest_path"
+  echo "Parsed $src_path to $dest_path"
+done
```

Comment:
I only see empty folders in my `localizations` folder

Reply:
Weird that mkdir isnt making them, even though its accessing them fine. Will look into this.

Reply:
Fixed in latest commit; re-requesting now.